MAINTAINERS

```diff
@@ -0,0 +1,5 @@
+Zain Asgar, Stanford University <zasgar@stanford.edu> (@zasgar)
+Michelle Nguyen, Gimlet Labs <michelle@gimletlabs.ai> (@aimichelle)
+Vihang Mehta, Gimlet Labs <vihang@gimletlabs.ai> (@vihangm)
+James Bartlett, Gimlet Labs <james@gimletlabs.ai> (@jamesmbartlett)
+Dom Del Nano, Crowdstrike <dom.delnano@crowdstrike.com> (@ddelnano)
```

Comment:
I recently found out that my name is not formally “Del Nano”. Only a few of my family members have a space in their name but mine doesn’t 🙃


```suggestion
Dom Delnano, Crowdstrike <dom.delnano@crowdstrike.com> (@ddelnano)
```

Reply:
Updated! That is good to know--thanks Dom!